src/test/java/jnr/posix/FileStatTest.java

```diff
@@ -157,4 +155,12 @@ public void filestatFailed() throws Throwable {
             }
         }
     }
+
+    @Test public void statIsWritable() throws Throwable {
+      String tempDirectory = System.getenv("TEMP");
+      FileStat st = posix.stat(tempDirectory);
+
+      // FIXME: This fails on JRuby, but succeeds here. Why?
```

Comment:
> Are you sure win8 and win10 have different values? I am not sure if this platform work is needed or not?

Yes, they do in fact. I ran it in a Win8.1 and a Win10 VM to test that part. It behaves differently w/ different JVM versions though.

Thanks for the suggestion. **But**, the weird thing is that the test below succeeds - i.e. when running `jnr-posix` in itself, the folder is considered writable. When jruby calls this code, it fails.